docs/blog/2022-07-16_minio-test/2022-07-16_minio-test.md

```diff
@@ -1,8 +1,338 @@
 ﻿---
 slug: minio
-title: HwameiStor Supports Minio
-authors: [Simon]
+title: HwameiStor Supports MinIO
+authors: [Simon, Michelle]
 tags: [Test]
 ---
 
-# HwameiStor supports Minio
\ No newline at end of file
+# HwameiStor Supports MinIO
+
+## MinIO Introduction
+
+MinIO is a high performance object storage solution with native support for Kubernetes deployment. It can provide distributed, S3-compatible, and multi-cloud storage service in public cloud, private cloud, and edge computing scenarios. MinIO is software-defined and open source under GNU AGPL v3. It can also run well on x86 and other standard hardware.
+
+![MinIO design](minio-design.png)
+
+MinIO is designed to meet private cloud's requirements for high performance, in addition to all required features of object storage. MinIO is easy to use, efficient, and performant in providing scalable cloud-native object storgae.
+
+MinIO works well in tradition object storage scenarios, such  as secondary storage, disaster recovery, and archiving。 It is also competitive in machine learning, big data, private cloud, hybrid cloud and other emering fileds, needless to mention data analysis, high performance workloads, cloud-native applications.
+
+### MinIO Architecture
+
+MinIO is designed for cloud-native architecture, so it can be run as a ligtweight container and managed by external orchestration tools like Kubernetes.

```

Comment:
```suggestion
MinIO is designed for the cloud-native architecture, so it can be run as a lightweight container and managed by external orchestration tools like Kubernetes.

```